### ISSUE TITLE
virtualenv_mod contains invalid python 3 code

### ISSUE THREAD

**Issue by geuben** [saltstack/salt#44863]
I'm trying to convert one of my applications over to python 3 and have come across an issue with the virtualenv_mod code.

https://github.com/saltstack/salt/blob/develop/salt/modules/virtualenv_mod.py#L320

```
        An exception occurred in this state: Traceback (most recent call last):
                File "/usr/lib/python2.7/dist-packages/salt/state.py", line 1723, in call
                  **cdata['kwargs'])
                File "/usr/lib/python2.7/dist-packages/salt/loader.py", line 1650, in wrapper
                  return f(*args, **kwargs)
                File "/var/cache/salt/minion/extmods/states/virtualenv_file.py", line 9, in managed
                  name=os.path.join(__salt__['virtualenv.get_site_packages'](virtualenv), name),
                File "/usr/lib/python2.7/dist-packages/salt/modules/virtualenv_mod.py", line 326, in get_site_packages
                  raise CommandExecutionError('{stdout}\n{stderr}'.format(**ret))
              CommandExecutionError:
                File "/tmp/tmpXmXdlc", line 1
                  from distutils import sysconfig; print sysconfig.get_python_lib()
                                                                 ^
              SyntaxError: invalid syntax
```

There's a fair chance I'm doing something wrong in my conversion but that is definitely not valid syntax for python 3 and changing it has no impact on python 2 compatibility

**Comment by gtmanfred** [comment]
Nice catch, would you mind submitting a PR to fix that to the 2017.7 branch?

https://docs.saltstack.com/en/latest/topics/development/contributing.html

Thanks,
Daniel

**Comment by pkruk** [comment]
Hi I add a fix for that, and create a pull request:)

**Comment by gtmanfred** [comment]
Awesome! Thank you very much!

**Comment by geuben** [comment]
Thanks, was AFK for a few days

**Comment by rallytime** [comment]
Closed via #44931
